auth/ENVOY.sh

```diff
@@ -25,12 +25,6 @@ fi
 
 ### SERVER CONFIGURATION
 
-# These steps are needed to enable local calls to the Cloud Datastore API.
-# This is required when the registry-server is run locally.
-
-# Optionally run this to update your application-default credentials.
-#gcloud auth application-default login
```

Comment:
This commented code gets credentials needed to call the PubSub API from non-Google environments. I don't mind taking it out here, but we should document this somewhere.

Reply:
I see it's still in LOCAL.sh - that works for me.

Reply:
Ack.